cmd/gapic-showcase/qualify.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"strconv"
+	"time"
+
+	trace "github.com/google/go-trace"
+	"github.com/spf13/cobra"
+
+	"github.com/googleapis/gapic-showcase/cmd/gapic-showcase/qualifier"
+)
+
+func init() {
+	var timestamp string
+	timestamp = time.Now().Format("20060102.150405")
+	trace.Trace("timestamp = %q", timestamp)
+
+	settings := &qualifier.Settings{
+		Timestamp:    timestamp,
+		Verbose:      Verbose,
+		ShowcasePort: 7469,
+	}
+	qualifyCmd := &cobra.Command{
+		Use:   "qualify",
```

Comment:
I'm actually reconsidering this command name now. I'm concerned it's not completely obvious what it does. Maybe `"check-generator"` would be clearer. WDYT? Other ideas?

If we decide to change it, I can change the package name to match.

Reply:
> But the usage would have a direct object

Ah, yes ok then that is reasonable. "check" to me isn't a strong/descriptive enough word. `<rhetorical>` What are we checking? That it exists? That it compiles? `</rhetorical>` I think "qualify-generator" is a better sounding command because it implies that some sort of validation and comparison of the generator against expected results happens.

Now, digging down into the the "direct object" you mention: is that a sub-command or just an argument? Is it a flag with accepted values (i.e. python, java, go, etc.)? Or free form?

Reply:
nvm I read the LongUsage, it's an exact arg. Thanks for adding that.